src/js/disability-benefits/526EZ/helpers.jsx

```diff
@@ -117,3 +137,125 @@ export const privateMedicalRecordsIntro = ({ formData }) => {
+  // Adds error message for state if it is blank and one of the following countries:
+  // USA, Canada, or Mexico
+  if (stateRequiredCountries.has(formData.treatmentCenterCountry)
+    && formData.treatmentCenterState === undefined) {
+    // TODO: enable once validation determined 
+    // && currentSchema.required.length) {
+    errors.treatmentCenterState.addError('Please select a state or province');
+  }
+  const hasAddressInfo = stateRequiredCountries.has(formData.treatmentCenterCountry)
+    // TODO: enable once validation determined 
+    // && !currentSchema.required.length
+    && typeof formData.treatmentCenterCity !== 'undefined'
+    && typeof formData.treatmentCenterStreet !== 'undefined'
+    && typeof formData.treatmentCenterPostalCode !== 'undefined';
+
+  if (hasAddressInfo && typeof formData.treatmentCenterState === 'undefined') {
+    errors.treatmentCenterState.addError('Please enter a state or province, or remove other address information.');
+  }
+
+  validatePostalCodes(errors, formData);
+}
+
+export const recordReleaseWarning = (
+  <div className="usa-alert usa-alert-warning no-background-image">
+    <span>Limiting consent means that your doctor can only share records that are directly related to your condition. This could add to the time it takes to get your private medical records.</span>
+  </div>
+);
+
+export const documentDescription = () => {
+  return (
+    <div>
+      <p>File upload guidelines:</p>
+      <ul>
+        <li>File types you can upload: .pdf, .jpeg, or .png</li>
+        <li>Maximum file size: 50 MB</li>
+      </ul>
+      <p><em>Large files can be more difficult to upload with a slow Internet connection</em></p>
+    </div>
+  );
+};
+
+export const additionalDocumentDescription = () => {
+  return (
+    <div>
+      <p>If you have other evidence, like lay or buddy statements, that you would like to submit, you can upload them here.</p>
+      <p>File upload guidelines:</p>
+      <ul>
+        <li>File types you can upload: .pdf, .jpeg, or .png</li>
+        <li>Maximum file size: 50 MB</li>
+      </ul>
+      <p><em>Large files can be more difficult to upload with a slow Internet connection</em></p>
+    </div>
+  );
+};
+
+const documentLabels = {
+  1: 'Discharge',
+  2: 'Marriage related',
+  3: 'Dependent related',
+  // 4: 'VA preneed form',
+  5: 'Letter',
+  6: 'Other'
+};
+
+const getVACenterName = (center) => center.treatment.treatmentCenterName;
+const getPrivateCenterName = (release) => release.privateRecordRelease.treatmentCenterName;
+
+const listifyCenters = (center, idx, list) => {
+  const getName = center.treatment ? getVACenterName : getPrivateCenterName;
+  const notLast = idx < (list.length - 1);
+  const justOne = list.length === 1;
+  const atLeastThree = list.length > 2;
+  return (
+    <strong key={idx}>
+      {`${notLast || justOne ? '' : ' and '}${getName(center)}${atLeastThree && notLast ? ', ' : ''}`}</strong>
+  );
+};
```

Comment:
Would something other than `listifyCenters` be a better fit here?

Reply:
:thinking: `transmogrifyPrivateAndVAMedicalCenterNamesToOxfordCommaCompliantGrammaticallyCorrectCommaDelineatedList()`?